src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml

```diff
@@ -1,582 +1,608 @@
+			<StaticResource x:Key="FilledPasswordBoxForegroundFocused" ResourceKey="OnSurfaceBrush" />
+			<!-- In this case the opacity is not applied to the brush directly - known issue for ContentElement -->
+			<StaticResource x:Key="FilledPasswordBoxForegroundDisabled" ResourceKey="OnSurfaceBrush" />
+			<StaticResource x:Key="FilledPasswordBoxForegroundOpacityDisabled" ResourceKey="LowOpacity" />
+
+			<StaticResource x:Key="FilledPasswordBoxPlaceholderForeground" ResourceKey="OnSurfaceVariantBrush" />
+			<StaticResource x:Key="FilledPasswordBoxPlaceholderForegroundPointerOver" ResourceKey="OnSurfaceVariantBrush" />
+			<StaticResource x:Key="FilledPasswordBoxPlaceholderForegroundFocused" ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="FilledPasswordBoxPlaceholderForegroundDisabled" ResourceKey="OnSurfaceLowBrush" />
+
+			<StaticResource x:Key="FilledPasswordBoxRevealButtonForeground" ResourceKey="OnSurfaceVariantBrush" />
+			<!--#endregion-->
+
+			<!--#region FilledPasswordBoxTypo-->
+			<StaticResource x:Key="FilledPasswordBoxFontFamily" ResourceKey="MaterialMediumFontFamily" />
+			<StaticResource x:Key="FilledPasswordBoxFontWeight" ResourceKey="BodyLargeFontWeight" />
+			<StaticResource x:Key="FilledPasswordBoxFontSize" ResourceKey="BodyLargeFontSize" />
+			<StaticResource x:Key="FilledPasswordBoxCharacterSpacing" ResourceKey="BodyLargeCharacterSpacing" />
+			<!--#endregion-->
+
+			<!--#region MaterialOutlinedPasswordBoxStyle-->
+			<StaticResource x:Key="OutlinedPasswordBoxBackground" ResourceKey="SystemControlTransparentBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxBackgroundPointerOver" ResourceKey="SystemControlTransparentBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxBackgroundFocused" ResourceKey="SystemControlTransparentBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
+
+			<StaticResource x:Key="OutlinedPasswordBoxBorderBrush" ResourceKey="OutlineBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxBorderBrushPointerOver" ResourceKey="OnSurfaceBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxBorderBrushFocused" ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxBorderBrushDisabled" ResourceKey="OnSurfaceDisabledLowBrush" />
+
+			<StaticResource x:Key="OutlinedPasswordBoxForeground" ResourceKey="OnSurfaceBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxForegroundPointerOver" ResourceKey="OnSurfaceBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxForegroundFocused" ResourceKey="OnSurfaceBrush" />
+			<!-- In this case the opacity is not applied to the brush directly - known issue for ContentElement -->
+			<StaticResource x:Key="OutlinedPasswordBoxForegroundDisabled" ResourceKey="OnSurfaceBrush" />
+			<StaticResource x:Key="OutlinedPasswordBoxForegroundOpacityDisabled" ResourceKey="LowOpacity" />
```

Comment:
Would it not be better to give an opacity for each state, so that someone using this style would not be limited to full colors for the foreground?

Reply:
Same thing for textbox

Reply:
I prefer to better investigate if the issue is coming from Uno or from the code-behind of the control from Microsoft.
Will revisit depending on the outcome. But prefer not to add a lot of resources for nothing if we will need to remove them if it is related to Uno only.